lua/entities/gmod_wire_expression2/core/custom/ftrace.lua

```diff
@@ -46,10 +46,10 @@ local varDefPrint = CreateConVar(gsVarPrefx.."_dprn", "TALK", gnServerControled,
 local gsFormLogs  = "E2{%s}{%d}:ftrace: %s" -- Contains the logs format of the addon
 local gsDefPrint  = varDefPrint:GetString() -- Default print location
 local gtPrintName = {} -- Contains the print location specification
-      gtPrintName["NOTIFY" ] = HUD_PRINTNOTIFY
-      gtPrintName["CONSOLE"] = HUD_PRINTCONSOLE
-      gtPrintName["TALK"   ] = HUD_PRINTTALK
-      gtPrintName["CENTER" ] = HUD_PRINTCENTER
+			gtPrintName["NOTIFY" ] = HUD_PRINTNOTIFY
+			gtPrintName["CONSOLE"] = HUD_PRINTCONSOLE
+			gtPrintName["TALK"   ] = HUD_PRINTTALK
+			gtPrintName["CENTER" ] = HUD_PRINTCENTER

```

Comment:
> **Can you get rid of the tabs/spaces completely?** We don't need to align stuff. Consistent tabbing is more important.

I think @thegrb93 meant remove the indentation all together, no? (imo looks cleaner this way)

```lua
local gtPrintName = {} -- Contains the print location specification
gtPrintName["NOTIFY"] = HUD_PRINTNOTIFY
gtPrintName["CONSOLE"] = HUD_PRINTCONSOLE
gtPrintName["TALK"] = HUD_PRINTTALK
gtPrintName["CENTER"] = HUD_PRINTCENTER
```

Reply:
Yes, that's what I meant.

Reply:
Fixed: d5ec8ed